conmon/conmon.c

```diff
@@ -452,6 +492,68 @@ static int write_k8s_log(int fd, stdpipe_t pipe, const char *buf, ssize_t buflen
 	return 0;
 }
 
+/*
+ * PROPOSED: CRI Stream Format, variable length file format
+ *
+ * %d-%02d-%02dT%02d:%02d:%02d.%09ld%c%02d:%02d %(stream)s %(buflen)d %(buf)s
```

Comment:
We should add the "byte" offset of the stream here where this buffer occurs so that the consumer on the other side does not have to rely on timestamps being different.